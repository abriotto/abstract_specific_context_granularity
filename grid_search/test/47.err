  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:01, 63.03it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:00<00:01, 61.68it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:00<00:01, 59.60it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:00<00:01, 66.52it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:00<00:00, 71.91it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:00<00:00, 65.79it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:00<00:00, 59.05it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:00<00:00, 58.88it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:01<00:00, 62.01it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:01<00:00, 66.74it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:01<00:00, 68.11it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:01<00:00, 70.39it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 63.68it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 74.98it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:00<00:00, 68.52it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:00<00:00, 64.95it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 69.34it/s]
