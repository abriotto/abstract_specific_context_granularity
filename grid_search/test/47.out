# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7106340527534485, "acc": 0.5014263987541199, "length": 3.533149003982544, "mode": "train", "epoch": 1}
{"loss": 0.7008417844772339, "acc": 0.509208083152771, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.7010236382484436, "acc": 0.5041998028755188, "length": 2.5487208366394043, "mode": "train", "epoch": 2}
{"loss": 0.6921669840812683, "acc": 0.4963167607784271, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.693518340587616, "acc": 0.515392005443573, "length": 1.3738356828689575, "mode": "train", "epoch": 3}
{"loss": 0.691863477230072, "acc": 0.511049747467041, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6769221425056458, "acc": 0.5903196930885315, "length": 1.8255031108856201, "mode": "train", "epoch": 4}
{"loss": 0.6423145532608032, "acc": 0.6509570479393005, "length": 2.276243209838867, "mode": "test", "epoch": 4}
{"loss": 0.6324326395988464, "acc": 0.6634871363639832, "length": 2.41192889213562, "mode": "train", "epoch": 5}
{"loss": 0.6071400046348572, "acc": 0.6800394058227539, "length": 2.4254143238067627, "mode": "test", "epoch": 5}
{"loss": 0.5966423153877258, "acc": 0.6708011627197266, "length": 2.3972442150115967, "mode": "train", "epoch": 6}
{"loss": 0.6260278224945068, "acc": 0.6611831188201904, "length": 2.3259668350219727, "mode": "test", "epoch": 6}
{"loss": 0.5802472233772278, "acc": 0.7037595510482788, "length": 2.4498486518859863, "mode": "train", "epoch": 7}
{"loss": 0.5990030765533447, "acc": 0.6990004181861877, "length": 2.430939197540283, "mode": "test", "epoch": 7}
{"loss": 0.5507002472877502, "acc": 0.7272858023643494, "length": 2.5587520599365234, "mode": "train", "epoch": 8}
{"loss": 0.586851954460144, "acc": 0.7002629637718201, "length": 2.5414364337921143, "mode": "test", "epoch": 8}
{"loss": 0.5263486504554749, "acc": 0.7591601014137268, "length": 2.791839122772217, "mode": "train", "epoch": 9}
{"loss": 0.5336225032806396, "acc": 0.7526419758796692, "length": 2.756906032562256, "mode": "test", "epoch": 9}
{"loss": 0.48116281628608704, "acc": 0.7928841710090637, "length": 2.956705331802368, "mode": "train", "epoch": 10}
{"loss": 0.5033936500549316, "acc": 0.7790647149085999, "length": 3.287292718887329, "mode": "test", "epoch": 10}
{"loss": 0.44110679626464844, "acc": 0.8044909238815308, "length": 3.19111704826355, "mode": "train", "epoch": 11}
{"loss": 0.4821396768093109, "acc": 0.797579288482666, "length": 3.154696226119995, "mode": "test", "epoch": 11}
{"loss": 0.4041736125946045, "acc": 0.8328628540039062, "length": 3.218003511428833, "mode": "train", "epoch": 12}
{"loss": 0.462613046169281, "acc": 0.8166362047195435, "length": 3.2707183361053467, "mode": "test", "epoch": 12}
{"loss": 0.37953993678092957, "acc": 0.8445123434066772, "length": 3.3512167930603027, "mode": "train", "epoch": 13}
{"loss": 0.44394823908805847, "acc": 0.8219614028930664, "length": 3.220994472503662, "mode": "test", "epoch": 13}
{"loss": 0.368834912776947, "acc": 0.8464754819869995, "length": 3.364251136779785, "mode": "train", "epoch": 14}
{"loss": 0.4290921688079834, "acc": 0.8334293961524963, "length": 3.3701658248901367, "mode": "test", "epoch": 14}
{"loss": 0.341738760471344, "acc": 0.8618875741958618, "length": 3.512514591217041, "mode": "train", "epoch": 15}
{"loss": 0.4475933015346527, "acc": 0.8250565528869629, "length": 3.569060802459717, "mode": "test", "epoch": 15}
{"loss": 0.32373401522636414, "acc": 0.8703960180282593, "length": 3.5609309673309326, "mode": "train", "epoch": 16}
{"loss": 0.41500312089920044, "acc": 0.8473389148712158, "length": 3.558011054992676, "mode": "test", "epoch": 16}
{"loss": 0.31680646538734436, "acc": 0.8721958994865417, "length": 3.5929248332977295, "mode": "train", "epoch": 17}
{"loss": 0.4121478199958801, "acc": 0.8367067575454712, "length": 3.70165753364563, "mode": "test", "epoch": 17}
{"loss": 0.3022890090942383, "acc": 0.8786666989326477, "length": 3.6090352535247803, "mode": "train", "epoch": 18}
{"loss": 0.4437423646450043, "acc": 0.8368062376976013, "length": 3.5414364337921143, "mode": "test", "epoch": 18}
{"loss": 0.29175183176994324, "acc": 0.8777801394462585, "length": 3.6305716037750244, "mode": "train", "epoch": 19}
{"loss": 0.4127138555049896, "acc": 0.846869707107544, "length": 3.878453016281128, "mode": "test", "epoch": 19}
{"loss": 0.2804478108882904, "acc": 0.88897705078125, "length": 3.763723850250244, "mode": "train", "epoch": 20}
{"loss": 0.4425583779811859, "acc": 0.8340814709663391, "length": 3.8618783950805664, "mode": "test", "epoch": 20}
{"loss": 0.2663308084011078, "acc": 0.8947741985321045, "length": 3.803652286529541, "mode": "train", "epoch": 21}
{"loss": 0.4239277243614197, "acc": 0.8449609279632568, "length": 3.9281768798828125, "mode": "test", "epoch": 21}
{"loss": 0.2550998330116272, "acc": 0.9002287983894348, "length": 3.8113040924072266, "mode": "train", "epoch": 22}
{"loss": 0.4274009168148041, "acc": 0.8314244151115417, "length": 3.889502763748169, "mode": "test", "epoch": 22}
{"loss": 0.2559618055820465, "acc": 0.8996862173080444, "length": 3.8186380863189697, "mode": "train", "epoch": 23}
{"loss": 0.45403119921684265, "acc": 0.8285219669342041, "length": 3.944751262664795, "mode": "test", "epoch": 23}
{"loss": 0.2429160624742508, "acc": 0.9075876474380493, "length": 3.8456544876098633, "mode": "train", "epoch": 24}
{"loss": 0.408171147108078, "acc": 0.8584596514701843, "length": 3.911602258682251, "mode": "test", "epoch": 24}
{"loss": 0.22967267036437988, "acc": 0.9136458039283752, "length": 3.8692119121551514, "mode": "train", "epoch": 25}
{"loss": 0.41476377844810486, "acc": 0.8421386480331421, "length": 3.966850757598877, "mode": "test", "epoch": 25}
{"loss": 0.22384844720363617, "acc": 0.9154231548309326, "length": 3.891070604324341, "mode": "train", "epoch": 26}
{"loss": 0.43381261825561523, "acc": 0.8545119166374207, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.21998654305934906, "acc": 0.9219995737075806, "length": 3.909226417541504, "mode": "train", "epoch": 27}
{"loss": 0.38971656560897827, "acc": 0.8655616641044617, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.19197455048561096, "acc": 0.9274742007255554, "length": 3.947021484375, "mode": "train", "epoch": 28}
{"loss": 0.411038339138031, "acc": 0.8489870429039001, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.19599208235740662, "acc": 0.9287519454956055, "length": 3.9413297176361084, "mode": "train", "epoch": 29}
{"loss": 0.4695279598236084, "acc": 0.8535911440849304, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.191212996840477, "acc": 0.9261685013771057, "length": 3.956216812133789, "mode": "train", "epoch": 30}
{"loss": 0.46347495913505554, "acc": 0.8508287072181702, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.18563023209571838, "acc": 0.9276251792907715, "length": 3.969508647918701, "mode": "train", "epoch": 31}
{"loss": 0.47038549184799194, "acc": 0.8609578609466553, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.19520041346549988, "acc": 0.9261382222175598, "length": 3.977179527282715, "mode": "train", "epoch": 32}
{"loss": 0.4279741048812866, "acc": 0.8692449331283569, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.17315131425857544, "acc": 0.9411056041717529, "length": 3.9760236740112305, "mode": "train", "epoch": 33}
{"loss": 0.4520084857940674, "acc": 0.8655617237091064, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.16918602585792542, "acc": 0.9351168274879456, "length": 3.982593536376953, "mode": "train", "epoch": 34}
{"loss": 0.4523346424102783, "acc": 0.860957682132721, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.15391092002391815, "acc": 0.9436274170875549, "length": 3.975512981414795, "mode": "train", "epoch": 35}
{"loss": 0.4515398442745209, "acc": 0.871086597442627, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.1495109498500824, "acc": 0.9446150660514832, "length": 3.983992099761963, "mode": "train", "epoch": 36}
{"loss": 0.47086986899375916, "acc": 0.8545119166374207, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.14939017593860626, "acc": 0.9493105411529541, "length": 3.9883015155792236, "mode": "train", "epoch": 37}
{"loss": 0.48521333932876587, "acc": 0.8572744131088257, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.14987772703170776, "acc": 0.9464446902275085, "length": 3.9851722717285156, "mode": "train", "epoch": 38}
{"loss": 0.46981081366539, "acc": 0.8554326891899109, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.14021272957324982, "acc": 0.9497250318527222, "length": 3.9931416511535645, "mode": "train", "epoch": 39}
{"loss": 0.45731696486473083, "acc": 0.8729282021522522, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.13861556351184845, "acc": 0.9475477337837219, "length": 3.991908311843872, "mode": "train", "epoch": 40}
{"loss": 0.46870920062065125, "acc": 0.8793737888336182, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.13652780652046204, "acc": 0.9481852054595947, "length": 3.993986129760742, "mode": "train", "epoch": 41}
{"loss": 0.43941670656204224, "acc": 0.8674033284187317, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.13235707581043243, "acc": 0.9557362198829651, "length": 3.9900288581848145, "mode": "train", "epoch": 42}
{"loss": 0.4715519845485687, "acc": 0.8812153935432434, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.14088477194309235, "acc": 0.948811411857605, "length": 3.9941248893737793, "mode": "train", "epoch": 43}
{"loss": 0.49845826625823975, "acc": 0.8637200593948364, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.14348146319389343, "acc": 0.9491783380508423, "length": 3.9963855743408203, "mode": "train", "epoch": 44}
{"loss": 0.468036025762558, "acc": 0.8766115307807922, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.13124246895313263, "acc": 0.9509953260421753, "length": 3.9954681396484375, "mode": "train", "epoch": 45}
{"loss": 0.4490744471549988, "acc": 0.8876611590385437, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.1239943653345108, "acc": 0.9562268853187561, "length": 3.995342969894409, "mode": "train", "epoch": 46}
{"loss": 0.47098320722579956, "acc": 0.8812154531478882, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.1320338249206543, "acc": 0.9519721865653992, "length": 3.9972405433654785, "mode": "train", "epoch": 47}
{"loss": 0.4553236961364746, "acc": 0.8756905794143677, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.12211700528860092, "acc": 0.9556393623352051, "length": 3.9967405796051025, "mode": "train", "epoch": 48}
{"loss": 0.4510093629360199, "acc": 0.8830571174621582, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.12028108537197113, "acc": 0.9562771916389465, "length": 3.997575283050537, "mode": "train", "epoch": 49}
{"loss": 0.4552593231201172, "acc": 0.8821362853050232, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.11162932962179184, "acc": 0.9605779051780701, "length": 3.997807264328003, "mode": "train", "epoch": 50}
{"loss": 0.390200138092041, "acc": 0.8950276374816895, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.10852627456188202, "acc": 0.961206316947937, "length": 3.9980545043945312, "mode": "train", "epoch": 51}
{"loss": 0.42203864455223083, "acc": 0.8922652006149292, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.10872270911931992, "acc": 0.9605982303619385, "length": 3.998378038406372, "mode": "train", "epoch": 52}
{"loss": 0.42062652111053467, "acc": 0.8941068649291992, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.10333523899316788, "acc": 0.9624419808387756, "length": 3.99863338470459, "mode": "train", "epoch": 53}
{"loss": 0.396692156791687, "acc": 0.8793738484382629, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.0982021689414978, "acc": 0.9652081727981567, "length": 3.9982869625091553, "mode": "train", "epoch": 54}
{"loss": 0.42850691080093384, "acc": 0.8913443684577942, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.09431352466344833, "acc": 0.9655072689056396, "length": 3.998173952102661, "mode": "train", "epoch": 55}
{"loss": 0.3911101222038269, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.09050650894641876, "acc": 0.9698329567909241, "length": 3.9988176822662354, "mode": "train", "epoch": 56}
{"loss": 0.37708163261413574, "acc": 0.8950278162956238, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.09416525810956955, "acc": 0.9639949798583984, "length": 3.998927116394043, "mode": "train", "epoch": 57}
{"loss": 0.4093831777572632, "acc": 0.892265260219574, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.09684091061353683, "acc": 0.9661546349525452, "length": 3.9992783069610596, "mode": "train", "epoch": 58}
{"loss": 0.4172394275665283, "acc": 0.8885819315910339, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.09378813952207565, "acc": 0.9679784774780273, "length": 3.9983017444610596, "mode": "train", "epoch": 59}
{"loss": 0.4299173355102539, "acc": 0.8848986625671387, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.09141861647367477, "acc": 0.9701418876647949, "length": 3.9986836910247803, "mode": "train", "epoch": 60}
{"loss": 0.41964611411094666, "acc": 0.8858195543289185, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.08920163661241531, "acc": 0.9679962992668152, "length": 3.9991307258605957, "mode": "train", "epoch": 61}
{"loss": 0.4352104365825653, "acc": 0.891344428062439, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.08535442501306534, "acc": 0.9683163166046143, "length": 3.999568462371826, "mode": "train", "epoch": 62}
{"loss": 0.4413352906703949, "acc": 0.889502763748169, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.08565119653940201, "acc": 0.965811550617218, "length": 3.9981143474578857, "mode": "train", "epoch": 63}
{"loss": 0.43998420238494873, "acc": 0.8950276374816895, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.08284962922334671, "acc": 0.9704612493515015, "length": 3.9992973804473877, "mode": "train", "epoch": 64}
{"loss": 0.41795891523361206, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.08471280336380005, "acc": 0.969237744808197, "length": 3.9996256828308105, "mode": "train", "epoch": 65}
{"loss": 0.40838631987571716, "acc": 0.901473343372345, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.07779523730278015, "acc": 0.9726207852363586, "length": 3.9995572566986084, "mode": "train", "epoch": 66}
{"loss": 0.432171106338501, "acc": 0.8941068649291992, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.08404301106929779, "acc": 0.9673932790756226, "length": 3.9996306896209717, "mode": "train", "epoch": 67}
{"loss": 0.44637560844421387, "acc": 0.8848987221717834, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.08671485632658005, "acc": 0.9716464877128601, "length": 3.9976963996887207, "mode": "train", "epoch": 68}
{"loss": 0.44002121686935425, "acc": 0.8968693017959595, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.07457182556390762, "acc": 0.9738085865974426, "length": 3.998664379119873, "mode": "train", "epoch": 69}
{"loss": 0.42432472109794617, "acc": 0.8931859731674194, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.07472918182611465, "acc": 0.9738418459892273, "length": 3.9993796348571777, "mode": "train", "epoch": 70}
{"loss": 0.4275660216808319, "acc": 0.8996317982673645, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.07132343202829361, "acc": 0.9763112664222717, "length": 3.999704599380493, "mode": "train", "epoch": 71}
{"loss": 0.4174526631832123, "acc": 0.8968692421913147, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.07021839916706085, "acc": 0.9738588333129883, "length": 3.9998536109924316, "mode": "train", "epoch": 72}
{"loss": 0.4618162512779236, "acc": 0.8885819911956787, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.07333620637655258, "acc": 0.97569739818573, "length": 3.999671220779419, "mode": "train", "epoch": 73}
{"loss": 0.45880696177482605, "acc": 0.8931859731674194, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.08202103525400162, "acc": 0.9692426919937134, "length": 3.999772071838379, "mode": "train", "epoch": 74}
{"loss": 0.4312666356563568, "acc": 0.8904235363006592, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.07339060306549072, "acc": 0.9732441902160645, "length": 3.999904155731201, "mode": "train", "epoch": 75}
{"loss": 0.4411657750606537, "acc": 0.8959483504295349, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.07895300537347794, "acc": 0.9732434153556824, "length": 3.999873399734497, "mode": "train", "epoch": 76}
{"loss": 0.45030373334884644, "acc": 0.8867403864860535, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.07014662027359009, "acc": 0.9775454998016357, "length": 3.9998152256011963, "mode": "train", "epoch": 77}
{"loss": 0.46884751319885254, "acc": 0.8931859731674194, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.08819602429866791, "acc": 0.9695541262626648, "length": 3.999880075454712, "mode": "train", "epoch": 78}
{"loss": 0.4253639578819275, "acc": 0.8858194947242737, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.08396434783935547, "acc": 0.9762436151504517, "length": 3.996687889099121, "mode": "train", "epoch": 79}
{"loss": 0.4275793433189392, "acc": 0.8987109065055847, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.08617297559976578, "acc": 0.9760127067565918, "length": 3.9999396800994873, "mode": "train", "epoch": 80}
{"loss": 0.4287291169166565, "acc": 0.8885818123817444, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.06973625719547272, "acc": 0.9766277074813843, "length": 3.9999260902404785, "mode": "train", "epoch": 81}
{"loss": 0.3998696804046631, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.07983215153217316, "acc": 0.9738564491271973, "length": 3.999725341796875, "mode": "train", "epoch": 82}
{"loss": 0.4244140088558197, "acc": 0.8931860327720642, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.08152981847524643, "acc": 0.9763055443763733, "length": 3.9993298053741455, "mode": "train", "epoch": 83}
{"loss": 0.43940186500549316, "acc": 0.8941068053245544, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.07877328246831894, "acc": 0.976626455783844, "length": 3.99989914894104, "mode": "train", "epoch": 84}
{"loss": 0.46832913160324097, "acc": 0.889502763748169, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.06941074877977371, "acc": 0.9787730574607849, "length": 3.999704122543335, "mode": "train", "epoch": 85}
{"loss": 0.41834521293640137, "acc": 0.8950276374816895, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.059551868587732315, "acc": 0.9784708023071289, "length": 3.999871253967285, "mode": "train", "epoch": 86}
{"loss": 0.45696574449539185, "acc": 0.890423595905304, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.06198565289378166, "acc": 0.9803187251091003, "length": 3.999969720840454, "mode": "train", "epoch": 87}
{"loss": 0.4716523587703705, "acc": 0.8766114711761475, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.06260284036397934, "acc": 0.9778586626052856, "length": 3.9999585151672363, "mode": "train", "epoch": 88}
{"loss": 0.46181246638298035, "acc": 0.8977901339530945, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.059849221259355545, "acc": 0.9778572916984558, "length": 3.999927043914795, "mode": "train", "epoch": 89}
{"loss": 0.4338192343711853, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.05780048668384552, "acc": 0.9799796938896179, "length": 3.9989545345306396, "mode": "train", "epoch": 90}
{"loss": 0.42926549911499023, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.05501292273402214, "acc": 0.9778586626052856, "length": 3.999936103820801, "mode": "train", "epoch": 91}
{"loss": 0.41703200340270996, "acc": 0.8977901339530945, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.06133268401026726, "acc": 0.9803179502487183, "length": 3.9999539852142334, "mode": "train", "epoch": 92}
{"loss": 0.4423357844352722, "acc": 0.8904234170913696, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.05546204373240471, "acc": 0.98365318775177, "length": 3.9987752437591553, "mode": "train", "epoch": 93}
{"loss": 0.4632996916770935, "acc": 0.8996317386627197, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.05685859173536301, "acc": 0.9778587818145752, "length": 3.999957323074341, "mode": "train", "epoch": 94}
{"loss": 0.41263970732688904, "acc": 0.9079190492630005, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.04823105037212372, "acc": 0.9855461716651917, "length": 3.999967098236084, "mode": "train", "epoch": 95}
{"loss": 0.4879457950592041, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.03951511159539223, "acc": 0.9898512959480286, "length": 3.9999613761901855, "mode": "train", "epoch": 96}
{"loss": 0.499146431684494, "acc": 0.8895029425621033, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.046025849878787994, "acc": 0.9861183166503906, "length": 3.9989373683929443, "mode": "train", "epoch": 97}
{"loss": 0.4754008650779724, "acc": 0.8876610994338989, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.035111937671899796, "acc": 0.9892371892929077, "length": 3.999990940093994, "mode": "train", "epoch": 98}
{"loss": 0.48093876242637634, "acc": 0.893186092376709, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.03732829540967941, "acc": 0.9898515939712524, "length": 3.9999775886535645, "mode": "train", "epoch": 99}
{"loss": 0.4734611511230469, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.06658299267292023, "acc": 0.9809345006942749, "length": 3.999990463256836, "mode": "train", "epoch": 100}
{"loss": 0.4621001183986664, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.777777910232544
